<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tkeFormDaysOpal</class>
 <widget class="QWidget" name="tkeFormDaysOpal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Дні опалення</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../tkeAbonent/tke/tke_rc">
    <normaloff>:/pictures/pictures/cube.png</normaloff>:/pictures/pictures/cube.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_Pidrozdil">
       <property name="text">
        <string>Підрозділ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_org_prev">
       <property name="text">
        <string>Попередній</string>
       </property>
       <property name="icon">
        <iconset resource="../tkeAbonent/tke/tke_rc">
         <normaloff>:/pictures/pictures/cubprev.png</normaloff>:/pictures/pictures/cubprev.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Pidrozdil_Name">
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_org_next">
       <property name="text">
        <string>Наступний</string>
       </property>
       <property name="icon">
        <iconset resource="../tkeAbonent/tke/tke_rc">
         <normaloff>:/pictures/pictures/cubnext.png</normaloff>:/pictures/pictures/cubnext.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_days_opal_lich">
       <property name="text">
        <string>Створити стрічку всього місяця</string>
       </property>
       <property name="icon">
        <iconset resource="../tkeAbonent/tke/tke_rc">
         <normaloff>:/pictures/pictures/iph.png</normaloff>:/pictures/pictures/iph.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView_dni_opal">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close_form">
       <property name="text">
        <string>Закрити</string>
       </property>
       <property name="icon">
        <iconset resource="../tkeAbonent/tke/tke_rc">
         <normaloff>:/pictures/pictures/button_cancel.png</normaloff>:/pictures/pictures/button_cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddString">
   <property name="icon">
    <iconset resource="../tkeAbonent/tke/tke_rc">
     <normaloff>:/pictures/pictures/greplus.png</normaloff>:/pictures/pictures/greplus.png</iconset>
   </property>
   <property name="text">
    <string>Додати стрічку</string>
   </property>
  </action>
  <action name="actionDelString">
   <property name="icon">
    <iconset resource="../tkeAbonent/tke/tke_rc">
     <normaloff>:/pictures/pictures/redminus.png</normaloff>:/pictures/pictures/redminus.png</iconset>
   </property>
   <property name="text">
    <string>Видалити стрічку</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../tkeAbonent/tke/tke_rc"/>
  <include location="../tkeAbonent/tke/tke_rc"/>
 </resources>
 <connections/>
</ui>
